data/sql/updates/pending_db_world/rev_1651987971178548000.sql

```diff
@@ -0,0 +1,5 @@
+--
+DELETE FROM `gossip_menu` WHERE `MenuID` IN (7106,7107);
+INSERT INTO `gossip_menu` VALUES (7106, 8363), (7107, 8364);
```

Comment:
Sniffed?

Reply:
Kind of. Korfax has gossip_menu_options with ActionMenuID=7106/7107, but there is no gossip menu with those IDs so it breaks and shows default text. All I did was add the gossip_menu entries it was looking for and linked them to existing npc_text.

Reply:
ok